src/extension/exponent/exponentHelper.ts

```diff
@@ -63,12 +63,12 @@ export class ExponentHelper {
             .then((user) => {
                 if (!user) {
                     let username = "";
-                    return showMessage("You need to login to exponent. Please provide username and password to login. If you don't have an account we will create one for you.")
+                    return showMessage("You need to login to Expo. Please provide username and password in the input boxes after closing this window. If you don't have an account, please go to https://expo.io to create one.")
```

Comment:
Small change in the message - `You need to login to Expo. Please provide your Expo account username and password in the input boxes after closing this window. If you don't have an account, please go to https://expo.io to create one.`

Also can we add a note in the debug section why the extension is asking for this information?

Reply:
@dhanvikapila yes, of course.

Reply:
@dhanvikapila Well, there is a caveat - actually, we need user to be logged in only in some case like when he is going to use tunelling (e.g. which allows devices outside of your LAN to access the the above servers without you needing to change your firewall settings) instead of LAN, or when he is going to publish his app to Expo. The thing is that is hard to detect such cases and, eventually, ask user to log in only when it needed, so that's why we are asking them to log in every time. I don't know how to correctly describe this in docs in terms of debugging instructions and user experience. Do you have any thoughts?